src/Raven.Server/Utils/Monitoring/ServerMetrics.cs

```diff
@@ -0,0 +1,234 @@
+// -----------------------------------------------------------------------
+//  <copyright file="ServerMonitoring.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using Raven.Client.ServerWide;
+using Raven.Server.Commercial;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Utils.Monitoring
+{
+    public class ServerMetrics
+    {
+        public string ServerVersion { get; set; }
+        public string ServerFullVersion { get; set; }
+        public int UpTimeInSec { get; set; }
+        public int ServerProcessId { get; set; }
+        public int CurrentNumberOfRunningBackups { get; set; }
+        
+        public ConfigurationMetrics Config { get; set; }
+        public CpuMetrics Cpu { get; set; }
+        public MemoryMetrics Memory { get; set; }
+        public DiskMetrics Disk { get; set; }
+        public LicenseMetrics License { get; set; }
+        public NetworkMetrics Network { get; set; }
+        public CertificateMetrics Certificate { get; set; }
+        public ClusterMetrics Cluster { get; set; }
+        public AllDatabasesMetrics Databases { get; set; }
+        
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(ServerVersion)] = ServerVersion,
+                [nameof(ServerFullVersion)] = ServerFullVersion,
+                [nameof(UpTimeInSec)] = UpTimeInSec,
+                [nameof(ServerProcessId)] = ServerProcessId,
+                [nameof(CurrentNumberOfRunningBackups)] = CurrentNumberOfRunningBackups,
```

Comment:
I'm wondering if it is not better to wrap this in Backups and put there not only this but also max number of concurrent backups?

Reply:
ok

Reply:
done